src/code/z_camera.c

```diff
@@ -4353,6 +4353,9 @@ s32 Camera_Subj3(Camera* camera) {
     return 1;
 }
 
+#define vCrawlSpaceBackPos temp1
+#define vPlayerDistToFront temp2
```

Comment:
Feel like these would be better inside the function, would flow better since they get swapped out half-way through.

Reply:
Also, out of curiosity, what is meant by the `v` prefix?

Reply:
its used in a few places in the codebase to indicate a variable alias

Reply:
Interesting, looks like the other places don't bother undefing them, just have multiple defines resolve to the same thing. Not sure if this should conform to that style?

Reply:
yea i dont think they particularly need to be undefined. the v prefix is intended to make them unique anyway, and if its changed to be scoped in the function there wont be any clashes either.
at the very least like you mentioned, if we do decide to start undefining them, its something we'd have to do in multiple places

Reply:
Do `#define` preprocessor commands follow scope? So if I define something inside the function, will it undefine internally once it's out of scope? I suppose I can just test it myself too

Reply:
No defines don't follow C scopes, the preprocessor is unaware of C syntax